repose-aggregator/tests/performance-tests/roles/gatling/files/gatling.conf

```diff
@@ -0,0 +1,132 @@
+#########################
+# Gatling Configuration #
```

Comment:
A fair concern. So if we weren't to copy the entire config file, how would we configure the parts we care about? That seems like it would be a post-installation step to modify the default, installed `gatling.conf` file. However, then it seems like we either need to have a tool that understands the format and allows us to modify parts, or take a more brutal approach with something like sed.

There might be a better approach that my braining isn't thinking of right now, though.